src/main/java/org/hpccsystems/jdbcdriver/HPCCResultSet.java

```diff
@@ -1,2330 +1,2331 @@
-                                metadatacols = new ArrayList();
-                                for (int datasetelementindex = 0; datasetelementindex < datasetelements.getLength(); datasetelementindex++)
-                                {
-                                    Node datasetelement = datasetelements.item(datasetelementindex);
-                                    if (datasetelement.getNodeName().equals("xs:element"))
-                                    {
-                                        NamedNodeMap attributes = datasetelement.getAttributes();
-                                        if (attributes != null)
-                                        {
-                                            Node typeattribute = attributes.getNamedItem("type");
-                                            if (typeattribute != null)
-                                            {
-                                                String typename = typeattribute.getNodeValue();
-                                                if (customTypeMap != null && customTypeMap.containsKey(typename))
-                                                    typename = customTypeMap.get(typename);
-                                                metadatacols.add(new HPCCColumnMetaData(attributes.getNamedItem("name").getNodeValue(), datasetelementindex, HPCCJDBCUtils.mapXSDTypeName2SQLtype(typename)));
-                                            }
-                                        }
-                                    }
-                                }
-                            }
-                        }
-                    }
-                    break;
-                }
-            }
-        }
-
-        if (resultMetadata == null)
-        {
-            if (metadatacols != null)
-                resultMetadata = new HPCCResultSetMetadata(metadatacols, tablename);
-            else
-                throw new Exception("Could not find result schema");
-        }
-
-        int rowcount = encapsulateDataSet(rowList);
-
-        long elapsedTime = System.currentTimeMillis() - startTime;
-        HPCCJDBCUtils.traceoutln(Level.INFO, "Finished Parsing results.");
-        HPCCJDBCUtils.traceoutln(Level.INFO, "Total elapsed http request/response time in milliseconds: " + elapsedTime);
-
-        return rowcount;
-    }
-
-    //Introduced in java 1.7 @Override
-    public <T> T getObject(int columnIndex, Class<T> type) throws SQLException
-    {
-        throw new UnsupportedOperationException("HPCCResultSet: getPseudoColumns Not supported yet.");
-    }
-
-    //Introduced in java 1.7 @Override
-    public <T> T getObject(String columnLabel, Class<T> type) throws SQLException
-    {
-        throw new UnsupportedOperationException("HPCCResultSet: getPseudoColumns Not supported yet.");
-    }
-}
+/*##############################################################################
```

Comment:
github shows this entire file changed, but I wasn't able to find any actual changes

Reply:
It looks like my code editor fixed the tabbing in that file. Ignoring white space changes there was a single change in that file on line 211: Changing GetResultsResponse to GetResultsResponseWrapper.